plugins/catalog-backend/src/service/router.ts

```diff
@@ -131,6 +140,14 @@ export async function createRouter(
       });
   }
 
+  if (locationAnalyzer) {
+    router.post('/analyze-location', async (req, res) => {
+      const input = await validateRequestBody(req, analyzeLocationSchema);
+      const output = await locationAnalyzer.generateConfig(input);
+      res.status(201).send(output);
```

Comment:
Just a `200` right? Nothing actually being created and we're not returning a `Location` header

Reply:
Done